core/frontend/src/test/scala/manual/PropertyErrorManualTest.scala

```diff
@@ -206,4 +211,45 @@ object PropertyErrorManualTest {
 //    ImmutableValue.isImmutable[CCB]
 //    ImmutableValue.isImmutable[CCD]
 //  }
+
```

Comment:
In this file all test cases are commented-out, because they don't compile and I use them to manually test compilation errors from properties.